common-design/sass/cd-header/_cd-search--inline.scss

```diff
@@ -0,0 +1,119 @@
+/**
+ * Common Design Search form in header.
+ */
+
+.cd-search--inline {
+  float: left;
+  border-left: 1px solid $cd-border-color;
+  border-right: 1px solid $cd-border-color;
+
+  .cd-search__form {
+    background: $cd-white-bg-color;
+    display: block;
+    left: unset;
+    position: relative;
+    right: unset;
+  }
+
+  .cd-search__form-inner {
+    padding: 0;
+  }
+
+  .cd-search__input {
+    -webkit-appearance: none;
+    background: $cd-white-bg-color;
+    border: 0;
+    border-radius: 0;
+    box-shadow: none;
+    color: $cd-default-text-color;
+    font-size: map-get($cd-font-sizes, large);
+    height: $cd-site-header-height;
+    height: calc(#{$cd-site-header-height} - 16px); //padding
+    padding: 0 45px 0 25px;
+    width: auto;
+  }
+
+  .cd-search__submit {
+    align-items: center;
+    background: $cd-white-bg-color;
+    border: 0;
+    border-left: 1px solid $cd-border-color;
+    color: $cd-dark-text-color;
+    display: flex;
+    font-size: map-get($cd-font-sizes, large);
+    font-weight: bold;
+    height: calc(#{$cd-site-header-height} - 16px); //padding
+    margin: 0;
+    padding: 0;
+    position: absolute;
+    right: $cd-container-padding;
+    text-transform: uppercase;
+    top: 0;
+
+    .icon-search {
+      font-size: 24px;
+      font-weight: bold;
+      padding: 0 10px;
+    }
+
+    &.js-has-focus {
+      background-color: $cd-highlight-red;
+    }
+
+  }
+
+  @media (min-width: map-get($cd-breakpoints, tablet)) {
```

Comment:
What do you think about nesting MQs inside the components themselves? I see how grouping by MQ in this case demonstrates the layout changes a bit better, but wanted to see what you thought about applying multiple breakpoints inside each component.

Reply:
I've gotten into the habit of adding breakpoints where needed inside the rule they apply to, as oppose to nesting all relevant rules in one breakpoint at the end of the file. As you say, I find it easier to read when everything is together.

I don't really mind. I'll follow whatever we decide to use going forward.